.travis.yml

```diff
@@ -4,9 +4,13 @@ node_js:
   - "4.3"
 
 before_install:
-  - "export CHROME_BIN=chromium-browser"
   - "export DISPLAY=:99.0"
   - "sh -e /etc/init.d/xvfb start"
+  - "sudo apt-get update"
```

Comment:
wouldn't you want to do `apt-get update && apt-get upgrade`. Is travis updating the chromium builds? Be nice if we didn't have to do this.